cmd/calyptia/create_aggregator_k8s.go

```diff
@@ -0,0 +1,135 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+
+	"github.com/spf13/cobra"
+	apiv1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+
+	cloud "github.com/calyptia/api/types"
+)
+
+var reHostnameRFC1123 = regexp.MustCompile(`^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$`)
```

Comment:
Probably all those custom cloud side validators should go into cloud/api

Reply:
🤔 
I just remove it completely.

This validation was required before, when I tried to pass the name to the core daemon, and this check was to avoid a later error discovery.

But now we create the aggregator before the k8s resources. So this validation is not required anymore ;)